app/CocoaPods.xcodeproj/project.pbxproj

```diff
@@ -543,6 +543,7 @@
 			baseConfigurationReference = C511D44E7F9B7A0C0EEF51D0 /* Pods.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CODE_SIGN_IDENTITY = "Developer ID Application";
```

Comment:
shouldn't this be done in the rakefile so you can still build in release, and only when building to distribute the cert is required?